test/CellDataTests/DiracDeltasTests.jl

```diff
@@ -37,6 +37,22 @@ degree = 2
 δ = DiracDelta{0}(model,tags=[2,4])
 @test sum(δ(v)) ≈ 5
 
+# Tests for DiracDelta at a generic Point in the domain #
+
+p = VectorValue(0.2,0.3)
+δ = DiracDelta(p,model)
+@test sum(δ(v)) == v(p)
+
+pvec = [p,π*p]
+δ = DiracDelta(pvec,model)
+@test sum(δ(v)) == sum(v(pvec))
+
+# below passes but cannot use FESpace and FEFunction here
```

Comment:
Not sure why this commented code is here. Should it be here or we can just remove it?

Reply:
This is a good test but as I mentioned in the comment, I cant use `FESpace` and `FEFunction` here, as they are down the hierarchy. If it is ok I can use `using Gridap` for the test

Reply:
>  I cant use FESpace and FEFunction here, as they are down the hierarchy. 

I dont understand this comment. Can't you just add `using Gridap.FESpaces` to the module in that test file?

Reply:
Yes I can.., my question was does `Gridap` `tests` also follow the module usage hierarchy as the `src`? If not I can do it.

Reply:
I dont think we have to constrain ourselves to the same module usage hierarchy. In Gridap this has to be respected because otherwise we may end with dependcy loops (cycles) among modules.

You can think as the test directory as an external program that is using Gridap tools and features. Thus, I do not see why we are forced to respect that constraint.

Reply:
The other thing that we have to think is whether `DiracDeltas.jl` should still be within `CellData` now given that we have introduced additional dependencies.

Reply:
> The other thing that we have to think is whether DiracDeltas.jl should still be within CellData now given that we have introduced additional dependencies.
No.. I haven't introduced any dependency in DiracDeltas.jl, used only the existing dependencies

Reply:
> No.. I haven't introduced any new dependency in DiracDeltas.jl, used only the existing dependencies

ok, I think you are right. Having a (new) reference to `CellField` (in the new `evaluate` function) does not imply a new module dependency, correct?

Reply:
Yes.. since `CellField` belongs to `CellData` module itself, it is not a new or external dependency.